libs/shared/console-shared/src/lib/flow-create-variable/ui/variable-row/variable-row.tsx

```diff
@@ -39,39 +49,77 @@ export function VariableRow(props: VariableRowProps) {
               message: 'Variable name cannot contain spaces.',
             },
           }}
-          render={({ field, fieldState: { error } }) => (
-            <InputTextSmall
-              className="mr-3 flex-1 shrink-0 grow"
-              name={field.name}
-              onChange={field.onChange}
-              value={field.value}
-              error={error?.message}
-              label="Variable"
-              errorMessagePosition="left"
-            />
-          )}
+          render={({ field, fieldState: { error } }) =>
+            watchReadOnly ? (
+              <div className="mr-3 flex items-center gap-2 truncate rounded border border-neutral-200 bg-neutral-100 px-2 text-sm text-neutral-350">
+                <span className="max-w-full truncate" title={field.value}>
+                  {field.value}
+                </span>
+                {watchDescription && (
+                  <Tooltip content={watchDescription}>
```

Comment:
This element should be at the right of the box 

<img width="194" alt="Screenshot 2024-07-30 at 16 30 10" src="https://github.com/user-attachments/assets/2d4d60b5-8c1e-42ef-8b52-8a88f5cce363">